ga4gh/cli.py

```diff
@@ -414,14 +414,22 @@ def __init__(self, args):
         self._referenceId = args.id
         self._start = args.start
         self._end = args.end
+        self._outputFormat = args.outputFormat
 
     def run(self):
-        iterator = self._client.listReferenceBases(
+        sequence = self._client.listReferenceBases(
             self._referenceId, self._start, self._end)
-        # TODO add support for FASTA output.
-        for segment in iterator:
-            print(segment, end="")
-        print()
+        if self._outputFormat == "text":
+            print(sequence)
+        else:
+            start = self._start if self._start else ""
+            end = self._end if self._end else ""
+            print(">{}:{}-{}".format(self._referenceId, start, end))
+
+            textWidth = 70
+            print(">{}".format(sequence[:textWidth-1]))
```

Comment:
Is this correct? I thought FASTAs just had the first line (with the ID) prefixed by '>'?